File: app/src/main/java/com/mjr/twaire/app/alleasy/fragment/physic/kinematics/vum/VUMSpeed3.java
package com.mjr.twaire.app.alleasy.fragment.physic.kinematics.vum;

import com.mjr.twaire.app.alleasy.fragment.physic.PhysicCalculations;
import com.mjr.twaire.app.alleasy.model.Data;

public class VUMSpeed3 extends PhysicCalculations {
    @Override
    protected void setTemplateAttributes() {
        getDatas().add(new Data("v = ", "m/s"));
        getDatas().add(new Data("a = ", "m/s2"));
        getDatas().add(new Data("∆t = ", "s"));
    }
    
    @Override
    public String getFragmentTag() {
        return getClass().getPackage().getName() + getClass().getName();
    }
}

File: app/src/main/java/com/mjr/app/alleasy/fragment/physics/kinematics/vum/VUMSpeed3.kt
package com.mjr.app.alleasy.fragment.physics.kinematics.vum

import com.mjr.app.alleasy.activity.AbstractPhysicalCalculation
import com.mjr.app.alleasy.model.Data

class VUMSpeed3 : AbstractPhysicalCalculation() {
    override fun setTemplateAttributes() {
        datas.apply {
            add(Data("v = ", "m/s"))
            add(Data("a = ", "m/s2"))
            add(Data("∆t = ", "s"))
        }
    }

    override fun onCalculateClickListener() {}
}